test-casskop-operator/test/obs-gap-2.yaml

```diff
@@ -0,0 +1,18 @@
+project: casskop-operator
+stage: test
+mode: observability-gap
+operator-pod-label: casskop-operator
+ce-name: sonar-cassandra-cluster
+ce-namespace: sieve
+ce-rtype: cassandracluster
+ce-diff-current: '{"spec": {"topology": {"dc": ["SONAR-SKIP", "SONAR-SKIP"]}}}'
+ce-diff-previous: '{"spec": {"topology": {"dc": ["SONAR-SKIP", "SONAR-SKIP", {"name":
+  "dc3", "rack": [{"name": "rack1"}], "nodesPerRacks": 0, "resources": {}}]}}}'
```

Comment:
I guess this obs-gap-2 is outdated. "nodesPerRacks" is 0 but in the above YAML it is 1. Also, the most recent one will not have a description.
@laphets Could you regenerate the obs-gap config for this case?